protocol-definitions/Client.yaml

```diff
@@ -11,18 +11,26 @@ methods:
       acquiresResource: false
       partitionIdentifier: -1
       params:
-        - name: username
+        - name: clusterName
           type: String
           nullable: false
           since: 2.0
+          doc: |
+            Cluster name that will client connect to.
```

Comment:
Minor: "...that client will connect to"?
Originally reported in:
https://github.com/hazelcast/hazelcast/pull/15772#discussion_r334929132